### ISSUE TITLE
Incorrect summary details for test run

### ISSUE THREAD

**Issue by isaacabraham** [haf/expecto#24]
I'm running a test group: -

```fsharp
let mathsTests =
    testList "Maths Tests" [
        testCase "2 * 2" <| fun _ -> Expect.equal (2 * 2) 4 "Should be equal"
        testCase "1 = 1" <| fun _ -> Expect.equal 1 2 "Should be equal"
    ]
```
```
[20:14:40 INF] EXPECTO? Running tests...
[20:14:40 ERR] Maths Tests/1 = 1 failed in 00:00:00.0079133. 
Should be equal. Actual value was 1 but had expected it to be 2.
  c:\Users\Isaac\Source\Repos\expecto-exp\src\Tests\Tests.fs(8,1): Tests.mathsTests@8-34.Invoke(String msg)

[20:14:40 INF] EXPECTO! 1 tests run in 00:00:00.0486770 - 1 passed, 0 ignored, 0 failed, 0 errored.
```

* The output shows that 1 test passed (correct) but incorrectly shows that 0 tests failed.
* The output shows that only 1 test was run, when 2 tests were run.

**Comment by Krzysztof-Cieslak** [comment]
Hmmm... are you using `2.1.0` ? It may be something I introduced when I was refactoring summary in #19

**Comment by isaacabraham** [comment]
Yeah, just got the latest version off nuget minutes before posting this.

**Comment by haf** [comment]
Validated – this is a major bug that got introduced – v2.1.0 has been pulled while @Krzysztof-Cieslak has a look.

**Comment by haf** [comment]
@Krzysztof-Cieslak Will you have a look at this today?

**Comment by isaacabraham** [comment]
@haf IMHO this is way less important than #23 - that one will screw up e.g. CI builds that are checking the return type.

**Comment by Krzysztof-Cieslak** [comment]
@haf, Yes will take a look at it today afternoon.

**Comment by haf** [comment]
Yes, #23 is why I unpublished the 2.1 nuget.

**Comment by haf** [comment]
I believe this is fixed with v2.1.1. Closing.
